<?xml version="1.0"?>
<launch>

    <include file="$(find multi_car_localization)/launch/fake_cars.launch" />

    <node pkg="multi_car_localization" type="fake_vicon.py" name="fake_vicon" output="screen">
    	<param name="num_cars" value="3" />
    </node>

</launch>